src/core/components/live-response.jsx

```diff
@@ -67,15 +69,15 @@ export default class LiveResponse extends React.Component {
                   </span>
                 }
                 {
-                  !body || isError ? null
```

Comment:
I'd like to see the ternary operators here be flipped around - the logical not in the comparison argument makes them hard to read.